### DISCUSSION TITLE
Swiping in mobile picture view?

### DISCUSSION THREAD

**Discussion by ezzra** [LycheeOrg/Lychee#2131] [Q&A] [answered]
Is there any way (with custom css, js or even code patching) to make swiping photos available on mobile devices? Pressing the button is really not convenient to present photos to my visitors, and then sometimes this happens:

![image](https://github.com/LycheeOrg/Lychee/assets/37742522/1a83bd21-783d-4951-a93d-1817baeac06b)

(and yes, the big border is also a problem here)

I am astonished, that lychee isn't more mobile friendly, thats where people use to watch photos when I send them a link..

**Comment by robs-fotoalbum** [comment]
The navigation on mobile is pretty cool to me. You swipe to the left or to the right edge, and on that edge you find then the navigation item with a small preview of the next slide. I think it's a nice combination of swipe and click.

The border is fine to me in this view. There is a full screen icon top right, if you click this, you get a large view of the photo. And in this view a pure swipe would be helpful, I think. In this case, the user is focussed on the photo and no navigation or other elements.

**Reply by ezzra** [reply]
> You swipe to the left or to the right edge

no swiping works for me, not in firefox or chrome on android, I need to tap the right corner, then the button appears and I need to tap again (and the button is sooo small, there is no sense to show a preview because anyway its not possible to recognise anything)

> There is a full screen icon top right

what icon? the monitor? it starts a slide show where I cannot do anything, just wait for the next photo

btw, Iam using Lychee 5.0.2

**Reply by robs-fotoalbum** [reply]
> no swiping works for me,

Could be a misunderstanding. Yes, a tap on the left and right edge bring the miniatures with the arrows. A swipe from the middle to the edge does the same. 

> what icon? the monitor? 

Yes, it brings the picture with maybe minimal smaller boarders and a blurred background. Now I have seen the slideshow, was not aware of it. Is there a control of the duration? 

I assume that there will be some additions made in the future, I don't know. We could suggest functionalities that we would like to see here and discuss with the developers what is possible in the future.

**Comment by ildyria** [comment]
> I am astonished, that lychee isn't more mobile friendly, thats where people use to watch photos when I send them a link..

Revert back to v4. It is way more mobile friendlier.

To be blunt, I haven't had time to dig into mobile friendliness on the v5 (basic functionalities work that was good enough). Also know that I was alone working on this front-end migration for the last year and a half. We are a very small team, new comers are always welcome!

We chose to release it now because:
- it was mature enough on laptop
- perfect is the enemy of done
- it allowed for a significantly improved user-management experience (which was a big standing request)
- the benefits of Livewire in term of ease of development would make it significantly easier to add more functionalities. Lets face it, Lychee has been close to stagnating for the last year.

And finally, yes it is in the plans, just haven't had time to dive into this.

**Comment by ildyria** [comment, marked as answer]
Implemented.
